website/docs/index.html.markdown

```diff
@@ -6,22 +6,22 @@ description: |-
   The Terraform Provider for VMware Cloud
 ---
 
-# terraform-provider-vmc
+# VMware Cloud on AWS Provider
+
+The VMware Cloud on AWS provider can be used to configure hybrid cloud infrastructure using the resources supported by VMware Cloud on AWS.
 
-The terraform-provider-vmc gives the VMC administrator a way to automate features
-of VMware Cloud on AWS using the VMC API.
 
 More information on VMC can be found on the [VMC Product
 Page](https://cloud.vmware.com/vmc-aws)
 
 Please use the navigation to the left to read about available data sources and
 resources.
 
-## Basic Configuration of the terraform-provider-vmc
+## Basic Configuration of the VMware Cloud on AWS Provider
 
-In order to use the terraform-provider-vmc you need to obtain the authentication
```

Comment:
Since the name is already fully described in the heading, you could refer to it as "the provider"